design-documents/graph-ql/coverage/customer-orders.md

```diff
@@ -79,55 +80,40 @@ The `id` will be a `base64_encode(increment_id)` which in future can be replaced
 
 ### Order Item
 
-The order items will be presented as separate interface which will have multiple implementations for invoice, shipment and credit memo types.
-
 ```graphql
-@doc("Interface to reprent order/invoice/shipment/credit memo items")
-interface SalesItemInterface {
+@doc("Order item")
+type OrderItemInterface {
```

Comment:
The name here should probably just be `OrderItem` since it's a `type`, right? Think it's just left over from a refactor

Reply:
Oh no, it should be interface instead of type, OrderItem type is defined below

Reply:
Ah right! Thanks